src/main/java/org/checkerframework/specimin/TargetMethodFinderVisitor.java

```diff
@@ -245,9 +245,11 @@ else if (!type.isUnknownType()) {
         }
 
         if (paramType.isReferenceType()) {
-          String paraTypeFullName =
-              paramType.asReferenceType().getTypeDeclaration().get().getQualifiedName();
-          updateUsedClassWithQualifiedClassName(paraTypeFullName);
+          if (paramType.asReferenceType().getTypeDeclaration().isPresent()) {
```

Comment:
Specimin should probably fail loudly (i.e., crash) if `getTypeDeclaration()` is not present, so I'm not sure that this change is a good idea. I'd approve it if you also added an `else` clause that throws a better exception than the `NoSuchElementException` that the code would throw otherwise, but I'd prefer to avoid Specimin silently failing.

Reply:
You are definitely right - silently failing is a bad idea. It might be beneficial to have a specific exception message to explain to the developer just exactly what is happening. But without running into this exception firsthand, I will rely on the default exception. Deleting this if statement.